lib/notification_service/notification_service/cli/commands/event_command.py

```diff
@@ -0,0 +1,134 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Event commands"""
+import functools
+from argparse import Namespace
+from datetime import datetime as dt
+from queue import Queue
+from typing import Callable, TypeVar, cast, List
+
+from notification_service.base_notification import EventWatcher, BaseEvent
+from notification_service.client import NotificationClient
+from notification_service.cli.simple_table import NotificationConsole
+
+
+T = TypeVar("T", bound=Callable)  # pylint: disable=invalid-name
+
+
+def action_logging(f: T) -> T:
```

Comment:
The function doesn't do any logging and it's recommended to rename this method.

Reply:
Renamed to `check_arguments`